### DISCUSSION TITLE
Allow to exclude repositories in GitHub App

### DISCUSSION THREAD

**Discussion by fhg-isi** [renovatebot/renovate#28618] [Suggest an Idea]
### Tell us more.

I installed renovate as GitHub App on organization level.
Currently is possible to allow repository access to "All repositories" or "Only selected repositories".

![image](https://github.com/renovatebot/renovate/assets/76424482/a861eae0-f1ff-44b2-8151-057ba8d073f4)

I suggest to extend the "all repositories" option with a selection to exclude distinct repositories. 
This way, if I add a new respositoriy, I do not need to manaully add it to the list of indvidual repositories, renovate should be applied for.

**Comment by rarkins** [comment]
This is a GitHub UI, not Renovate. Only GitHub has control over it.

**Reply by fhg-isi** [reply]
Thank you for clarification. 
a) What would be the right place to suggest it? 
b) Just found the config option:
https://docs.renovatebot.com/configuration-options/#enabled
However, it does not seem to have an effect when using renovate App on organization level. 
```
{
  "enabled": false
}
```

**Reply by rarkins** [reply]
You need to suggest it to GitHub, such as in their community forums

**Reply by fhg-isi** [reply]
https://github.com/orgs/community/discussions/120980
